UICatalog/UICatalog.cs

```diff
@@ -498,7 +498,8 @@ public UICatalogTopLevel ()
                 Title = "Categories",
                 BorderStyle = LineStyle.Single,
                 SuperViewRendersLineCanvas = true,
-                Source = new ListWrapper (_categories)
+                Source = new ListWrapper (_categories),
+                ScrollBarType = ScrollBarType.Both
```

Comment:
Why `Enabled`? This makes sense if it was deactivated before, being visible, but without the user being able to interact with it. For me, just one statement is simpler, such as `CategoryList.Padding.ScrollBarType = ScrollBarType.Both;` in case we want to put it in Padding, of course.

Reply:
> Why `Enabled`? This makes sense if it was deactivated before, being visible, but without the user being able to interact with it. For me, just one statement is simpler, such as `CategoryList.Padding.ScrollBarType = ScrollBarType.Both;` in case we want to put it in Padding, of course.

Because `Enabled` is perfectly suited for this, and already exists. Why add (or not remove) API surface area if there's already a primitive that is there?

Reply:
For me `Enabled=true` is toggle from grayed to normal color, that why I asked. If I remember when I created that property was for that propose.

Reply:
Ahh... Yes. the docs say "Gets or sets a value indicating whether this <see cref="Responder"/> can respond to user interaction."

I don't think we should change that. Your question is great.

We need two things:

1) A way for a `View` subclass to say "Scrollbars make no sense for me." I was thinking `Enabled` would be good for that. It's not. 

Ideas for what this mechanism could be:

- On `View`: `public ScrollBar HorizontalScrollBar` - If the property `is null` then duh. The question here is who's responsible for creating the `ScrollBar` instance? Lots of choices.
- On `View`: `public ScrollBarType ScrollBarType` - yuk. I just don't like adding more and more enums.

Other ideas?

2) A way for a user of a `View` subclass (or the subclass itself) to say "make the the H or V scrollbar be visible".

This should be the `Visible` property on `ScrollBar`. For example `_myView.HorizontalScrollBar.Visible = true`.